pages/common/zoxide.md

```diff
@@ -0,0 +1,28 @@
+# zoxide
+
+> Keep track of the directories you use most frequently, and use a ranking algorithm to navigate to the best match.
+> More information: <https://github.com/ajeetdsouza/zoxide>.
+
+- Go to the highest-ranked directory that contains "foo" in the name:
+
+`zoxide query {{foo}}`
+
+- Go to the highest-ranked directory that contains "foo" and then "bar":
+
+`zoxide query {{foo}} {{bar}}`
+
+- Interactive directory search (requires [fzf](https://github.com/junegunn/fzf)):
+
+`zoxide query --interactive`
+
+- Add a directory or increment it's rank:
```

Comment:
Perhaps this will read better?
```suggestion
- Add a directory or increment its rank:
```

`it's` ->  `it is`. `its` doesn't have an apostrophe if it's possessive :-)

Reply:
Absolutely correct, very lazy of me!